package.json

```diff
@@ -102,27 +102,25 @@
   "scripts": {
     "vscode:prepublish": "tsc -p ./",
     "compile": "tsc -watch -p ./",
-    "postinstall": "node ./node_modules/vscode/bin/install",
     "prettier": "prettier --write \"**/*.{js,jsx,ts,tsx,css}\""
   },
   "devDependencies": {
     "@types/node": "^10.12.2",
+    "@types/vscode": "^1.63.1",
     "husky": "^1.1.3",
     "lint-staged": "^8.0.4",
     "prettier": "^1.14.3",
-    "typescript": "^3.1.6",
-    "vscode": "^1.1.18"
+    "typescript": "^3.1.6"
   },
   "dependencies": {
     "@textlint/ast-node-types": "^4.0.3",
     "@textlint/ast-traverse": "^2.0.9",
     "@textlint/kernel": "^3.0.0",
     "@textlint/textlint-plugin-markdown": "^5.0.1",
     "@textlint/textlint-plugin-text": "^4.0.1",
-    "@types/p-queue": "^3.0.0",
     "@types/structured-source": "^3.0.0",
-    "p-queue": "^3.0.0",
-    "say": "^0.15.0",
+    "p-queue": "^6.6.2",
+    "say": "MatchaChoco010/say.js#make-3rd-party-voice-available",
```

Comment:
I've recommened to publish it as package.
Git's Branch is not immutable.

Scoped package is useful for fork repository.

- [How to Create and Publish a Scoped NPM Package (@you/package) from Scratch | by David Good | Medium](https://medium.com/@davidagood/how-to-create-and-publish-a-scoped-npm-package-you-package-from-scratch-30c5802b2df0)

Reply:
Ok, I will publish this fork repository as scoped package.